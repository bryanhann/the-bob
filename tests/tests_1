info "BEGIN TESTING THE ENVIRONMENT"

info "found: \$TEMP is ${TEMP}"
info "found: \$TMP is ${TMP}"

runtest \
	"\$TMP is \$TEMP" \
	"[ .$TMP == .$TEMP  ]"\
	0

runtest \
	"\$TMP is directory" \
	"testdir $TMP"\
	0

info "BEGIN TESTING THE TESTING FRAMEWORK"

runtest\
	"test return_with()" \
	"return_with 42"\
	42

