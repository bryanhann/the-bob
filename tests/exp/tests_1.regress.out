[info] BEGIN TESTING THE ENVIRONMENT
[info] found: $TEMP is
[info] found: $TMP is
[....] $TMP is $TEMP[ ok ]
[....] $TMP is directory[fail]
[....] $HOME/.local/bin is directory[fail]
[info] BEGIN TESTING THE TESTING FRAMEWORK
[....] test return_with()[ ok ]
